boot/freeldr/freeldr/arch/i386/xboxvideo.c

```diff
@@ -180,6 +183,12 @@ XboxVideoGetBufferSize(VOID)
   return (ScreenHeight - 2 * TOP_BOTTOM_LINES) / CHAR_HEIGHT * (ScreenWidth / CHAR_WIDTH) * 2;
 }
 
+VOID
+XboxVideoGetFontsFromFirmware(PULONG RomFontPointers)
+{
+    TRACE("XboxVideoGetFontsFromFirmware(): UNIMPLEMENTED\n");
```

Comment:
Not implemented or not available?

Reply:
Not sure, but I think it can be somehow **hack**plemented, so that's it.